src/test/ui/parser/tag-variant-disr-non-nullary.stderr

```diff
@@ -1,17 +1,12 @@
-error: discriminator values can only be used with a field-less enum
-  --> $DIR/tag-variant-disr-non-nullary.rs:2:11
+error[E0658]: discriminator values can only be used with a field-less enum
+  --> $DIR/tag-variant-disr-non-nullary.rs:2:5
    |
 LL |     Red = 0xff0000,
-   |           ^^^^^^^^ only valid in field-less enums
-LL |
-LL |     Green = 0x00ff00,
```

Comment:
How hard would it be to continue to highlight the other variants with the assigned discriminator values?

(Or was it a deliberate choice here to limit attention to the first occurrence? If so, why?)

Reply:
(Is this an artifact of invoking the feature gate diagnostic machinery?)

Reply:
(it seems like your other tests did not run into a similar issue. Maybe I'll look more closely to figure out why this is occurring here.)

Reply:
ah you explicitly noted this here: https://github.com/rust-lang/rust/pull/60732/files#r283095874